src/cortxfs/cortxfs_ops.c

```diff
@@ -115,8 +115,8 @@ int cfs_getattr(struct cfs_fs *cfs_fs, const cfs_cred_t *cred,
 	return rc;
 }
 
-int cfs_setattr(struct cfs_fs *cfs_fs, cfs_cred_t *cred, cfs_ino_t *ino,
-		struct stat *setstat, int statflag)
+static int __cfs_setattr(struct cfs_fs *cfs_fs, cfs_cred_t *cred,
```

Comment:
Lets have "inline" here to avoid the function call.

Reply:
fixed.